### ISSUE TITLE
App crash: Couldn't create Paper dir ...EventDatabase

### ISSUE THREAD

**Issue by Matej-Hlatky** [exponea/exponea-android-sdk#16]
Hello,
Android app crashes, when using Exponea SDK -- Couldn't create Paper dir:
```
Fatal Exception: java.lang.RuntimeException: Couldn't create Paper dir: /data/user/0/XXXX/files/EventDatabase
       at io.paperdb.DbStoragePlainFile.createPaperDir(DbStoragePlainFile.java:325)
       at io.paperdb.DbStoragePlainFile.assertInit(DbStoragePlainFile.java:316)
       at io.paperdb.DbStoragePlainFile.deleteIfExists(DbStoragePlainFile.java:211)
       at io.paperdb.Book.delete(Book.java:116)
       at com.exponea.sdk.database.ExponeaDatabaseImpl.remove(ExponeaDatabaseImpl.kt:57)
       at com.exponea.sdk.repository.EventRepositoryImpl.remove(EventRepositoryImpl.kt:27)
       at com.exponea.sdk.manager.FlushManagerImpl.onEventSentSuccess(FlushManagerImpl.kt:115)
       at com.exponea.sdk.manager.FlushManagerImpl.access$onEventSentSuccess(FlushManagerImpl.kt:12)
       at com.exponea.sdk.manager.FlushManagerImpl$trackCustomer$1.invoke(FlushManagerImpl.kt:95)
       at com.exponea.sdk.manager.FlushManagerImpl$trackCustomer$1.invoke(FlushManagerImpl.kt:12)
       at com.exponea.sdk.util.ExtensionsKt$enqueue$1.onResponse(Extensions.kt:22)
       at okhttp3.RealCall$AsyncCall.execute(RealCall.java:153)
       at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
       at java.lang.Thread.run(Thread.java:764)
```

Exponea SDK versions:  v1.1.7 and v2.2.3; both versions uses Paperdb v2.6
Android versions: 4.4.4, 5.1, 8.0.0

Cannot reproduce; error happening to end users.

**Comment by ArkhipenkaPiotr** [comment]
Isn't any progress with this issue?

**Comment by Matej-Hlatky** [comment]
Issue happen when calling `Exponea.anonymize()`.

**Comment by Igi4** [comment]
@Matej-Hlatky Hey, I am very sorry for the later reply. We'll try to provide you with more information as soon as possible.

Does that crash happen spontaneously at random times or on e.g. app start?

**Comment by Matej-Hlatky** [comment]
Hi @Igi4 ,

For the last 90 days (limit from Crashlytics) this occur only 5 times.

In all cases, this was after user logs-out from the app. In this process, we clear our internal preferences (stored user details), and start with `MainActivity`. However during this process, `Exponea.anonymize()` gets called twice, so this could be the issue.

I'll try to prevent this, update to latest Exponea SDK and let you know.

**Comment by wassil** [comment]
Fixed in 2.8.3 https://github.com/exponea/exponea-android-sdk/releases/tag/2.8.3
